scripts/configure-installer-tests-cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Setup to find nessasary data from cluster setup
+## Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+# Overrideable information
+AUTH_DIR=${AUTH_DIR:-"$(pwd)/auth"}
+KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+KUBEADMIN_PASSWORD_FILE_NAME=${KUBEADMIN_PASSWORD_FILE_NAME:-"kubeadmin-password"}
+KUBECONFIG_FILE_NAME=${KUBECONFIG_FILE_NAME:-"kubeconfig"}
+
+# CALCULATED INFORMATION
+KUBEADMIN_PASSWORD_FILE="${AUTH_DIR}/${KUBEADMIN_PASSWORD_FILE_NAME}"
+
+# Exported to current env
+export KUBECONFIG="${AUTH_DIR}/${KUBECONFIG_FILE_NAME}"
+
+# List of users to create
+USERS="developer odonoprojectattemptscreateproject odosingleprojectattemptscreate odologinnoproject odologinsingleproject1"
```

Comment:
User ```odonoprojectattemptscreate``` is missing. See https://github.com/redhat-developer/odo/blob/master/tests/e2e/e2e_test.go#L71 for reference

Reply:
Fixed